config/config_test.go

```diff
@@ -1042,11 +1040,14 @@ func TestInvalidEnforcePurpose(t *testing.T) {
 	cfg.GDPR.TCF2.Purpose10.EnforcePurpose = "invalid3"
 
 	errs := cfg.validate(v)
-	assert.Len(t, errs, 4)
-	assert.EqualError(t, errs[0], "gdpr.tcf2.purpose1.enforce_purpose must be no or full. Got ")
-	assert.EqualError(t, errs[1], "gdpr.tcf2.purpose5.enforce_purpose must be no or full. Got invalid1")
-	assert.EqualError(t, errs[2], "gdpr.tcf2.purpose6.enforce_purpose must be no or full. Got invalid2")
-	assert.EqualError(t, errs[3], "gdpr.tcf2.purpose10.enforce_purpose must be no or full. Got invalid3")
+
+	expectedErrs := []error{
+		fmt.Errorf("gdpr.tcf2.purpose1.enforce_purpose must be \"no\" or \"full\". Got "),
```

Comment:
Nitpick: Should just be `errors.New` since there is no formatting being performed.